sample-config-files/basic_config.yaml

```diff
@@ -0,0 +1,11 @@
+# YAML basic configuration file with examples of how to specify various types
+# of parameters.
+
+# Example specifying a string parameter
+outputpath: example_output
+
+# Example specifying a list parameter
+baselines: [gmail, calendar, groups, chat, drive, meet, sites, commoncontrols]

```

Comment:
It'd be helpful to know if arguments need to be denoted as a string or list type. 

For example the following will fail: 
```
baselines: gmail, calendar, groups, chat, drive, meet, sites, commoncontrols
or
b: *
```

Reply:
Good point. I added a comment that hopefully clarifies it.